tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -43,6 +43,24 @@ Feature: Module
     Given module ps_featuredproducts has following infos:
       | enabled | true |
 
+  Scenario: Reset module status
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | enabled        | true |
+      | installed      | true |
+    When I reset module "ps_featuredproducts"
```

Comment:
This step `Given module ps_featuredproducts has following infos:` is only used to check the status of the module, it only reads data so it won't modify the module state and should only be used to check the status after your reset action
We can't really test what the reset does but you can at least validate that it is still installed and enabled

```suggestion
    When I reset module "ps_featuredproducts"
    Then module ps_featuredproducts has following infos:
      | technical_name | ps_featuredproducts |
      | enabled        | true |
      | installed      | true |
```